[MON-21088] - [Documentation] Enhance database mysql plugin sql string (#2712)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mssql.md

2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.

<Tabs groupId="sync">
<TabItem value="Backup-Age" label="Backup-Age">

| Macro                            | Description                                                                                                                                | Valeur par défaut | Obligatoire |
|:---------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| FILTERNAME                       | Filter databases by name                                                                                                                   | ^(?!(tempdb))     |             |

> | UNIT                             | Select the unit for expires threshold. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is days | d                 |             |

Reviewer: Expiration thresholds ?

> | UNIT                             | Select the unit for expires threshold. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is days | d                 |             |

Reviewer: ```suggestion
| UNIT                             | Select the time unit for thresholds. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is days | d                 |             |
```

| CRITICALLOGFILESSPACEUSAGEFREE  | Thresholds                                                                                          |                   |             |
| WARNINGLOGFILESSPACEUSAGEPRCT   | Thresholds                                                                                          |                   |             |
| CRITICALLOGFILESSPACEUSAGEPRCT  | Thresholds                                                                                          |                   |             |
| EXTRAOPTIONS                    | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose         |             |

</TabItem>
<TabItem value="Deadlocks" label="Deadlocks">

| Macro        | Description                                                                                         | Valeur par défaut | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER       | Filter database to check                                                                            |                   |             |

Author: Because of this :
```    foreach my $row (@{$query_result}) {
        next if (defined($self->{option_results}->{filter_database}) && $self->{option_results}->{filter_database} ne ''
                    && $$row[0] !~ /$self->{option_results}->{filter_database}/);
        $self->{deadlocks}->{value} += $$row[1];
    }
```
I think it is a database name filter

> | FILTER       | Filter database to check                                                                            |                   |             |

Reviewer: Filter by database name?

> | FILTER       | Filter database to check                                                                            |                   |             |

Reviewer: ```suggestion
| FILTER       | Filter by database name                                                                            |                   |             |
```

| CRITICALJOBSTOTAL    | Thresholds                                                                                                                         |                   |             |
| WARNINGSTATUS        | Define the conditions to match for the status to be WARNING. You can use the following variables: %{name}, %{status}, %{duration}  |                   |             |
| CRITICALSTATUS       | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %{name}, %{status}, %{duration} |                   |             |
| EXTRAOPTIONS         | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                | --verbose         |             |

</TabItem>
<TabItem value="Locks-Waits" label="Locks-Waits">

| Macro        | Description                                                                                         | Valeur par défaut | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER       | Filter database to check                                                                            |                   |             |

Author: Same as above, I think name filter

> | FILTER       | Filter database to check                                                                            |                   |             |

Reviewer: ```suggestion
| FILTER       | Filter by database name                                                                            |                   |             |
```

| EXTRAOPTIONS                  | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose         |             |

</TabItem>
</Tabs>

3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.

## Comment puis-je tester le plugin et que signifient les options des commandes ?

Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
de commande depuis votre collecteur Centreon en vous connectant avec
l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
que le connecteur arrive bien à superviser une ressource en utilisant une commande
telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :

```bash
/usr/lib/centreon/plugins/centreon_mssql.pl \
	--plugin database::mssql::plugin \
	--hostname 10.0.0.1 \
	--port 1433 \

> 	--username 'USERNAME' \

Author: Je n'ai pas trouvé de raison précise donc on peut le remettre je pense

> 	--username 'USERNAME' \

Reviewer: ```suggestion
	--username 'USERNAME@DOMAIN' \
```

| --check-underlying-disk       | Check and consider underlying disk space for data and log files.                                                                                                                             |
| --ignore-unlimited            | Thresholds not applied on unlimited autogrowth data and log files.                                                                                                                           |
| --warning-* --critical-*      | Thresholds. Can be: 'datafiles-space-usage', 'datafiles-space-usage-free', 'datafiles-space-usage-prct' 'logfiles-space-usage', 'logfiles-space-usage-free', 'logfiles-space-usage-prct'.    |

</TabItem>
<TabItem value="Deadlocks" label="Deadlocks">

| Option               | Description                                           |
|:---------------------|:------------------------------------------------------|
| --warning-deadlocks  | Warning threshold number of dead locks per second.    |
| --critical-deadlocks | Critical threshold number of dead locks per second.   |

> | --filter-database    | Filter database to check.                             |

Reviewer: ```suggestion
| --filter-database    | Filter by database name.                             |
```

| --add-duration-perfdata  | Display job duration time.                                                                                                           |
| --warning-status         | Define the conditions to match for the status to be WARNING. You can use the following variables: %{name}, %{status}, %{duration}    |
| --critical-status        | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %{name}, %{status}, %{duration}   |
| --warning-* --critical-* | Thresholds. Can be: 'jobs-total', 'jobs-failed', 'jobs-success', 'jobs-canceled', 'jobs-running', 'jobs-retry'.                      |

</TabItem>
<TabItem value="Locks-Waits" label="Locks-Waits">

| Option                | Description                                           |
|:----------------------|:------------------------------------------------------|
| --warning-lockswaits  | Warning threshold number of lock-waits per second.    |
| --critical-lockswaits | Critical threshold number of lock-waits per second.   |

> | --filter-database     | Filter database to check.                             |

Reviewer: ```suggestion
| --filter-database     | Filter by database name.                             |
```
